lib/api_hammer/faraday/request_logger.rb

```diff
@@ -22,6 +22,11 @@ def initialize(app, logger, options={})
         @app = app
         @logger = logger
         @options = options
+        @log_tags = nil
+      end
+
+      def log_tags
+        @log_tags || 'DEFAULT_LOG_TAG'
```

Comment:
i did not know what it was, I just caught that it was undefined... 
I saw your fix on your other PR